src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateOnlyConverter.cs

```diff
@@ -9,30 +9,27 @@
 namespace System.ComponentModel
 {
     /// <summary>
-    /// Provides a type converter to convert <see cref='System.DateOnly'/>
-    /// objects to and from various other representations.
+    /// Provides a type converter to convert <see cref='System.DateOnly'/> objects to and from various other representations.
     /// </summary>
     public class DateOnlyConverter : TypeConverter
     {
+        /// <inheritdoc />
```

Comment:
(nit), it might just be me, but I always think the `summary` comment should go first. So I guess I'd expect this to look like

```suggestion
        /// <summary>
        /// Gets a value indicating whether this converter can convert an object in the given source type to a <see cref='System.DateOnly'/>
        /// object using the specified context.
        /// </summary>
        /// <inheritdoc />
```

Reply:
I'll apply the suggestion but I am seeing the pattern is `/// <inheritdoc />` is always listed first.

Reply:
never mind. I see in the doc you sent has the summary listed before the `inherit`.